pom.xml

```diff
@@ -130,9 +130,9 @@
     <glusterfs-hadoop.version>2.3.13</glusterfs-hadoop.version>
     <grpc.version>1.37.0</grpc.version>
     <gson.version>2.8.9</gson.version>
-    <netty.version>4.1.52.Final</netty.version>
+    <netty.version>4.1.86.Final</netty.version>
     <rocksdb.version>7.0.3</rocksdb.version>
-    <hadoop.version>3.3.1</hadoop.version>
+    <hadoop.version>3.2.3</hadoop.version>
```

Comment:
wouldn't this be a downgrade?

since our filesystem API is based on this hadoop version, this type of change could  have unintended consequences that may not be surfaced at the PR level. this needs to be separately scrutinized and would similarly advise excluding from this PR (along with the other hadoop version change)

Reply:
see https://github.com/Alluxio/alluxio/pull/17002